src/legacy/core_plugins/embeddable_api/public/np_ready/public/lib/embeddables/embeddable_factory.ts

```diff
@@ -53,28 +56,27 @@ export abstract class EmbeddableFactory<
   >,
   TSavedObjectAttributes extends SavedObjectAttributes = SavedObjectAttributes
 > {
+  // TODO: Rename this to `id` to align with other objects?
```

Comment:
action, embeddable and embeddable_factory all use `type`.  I think `type` is better because I think for dynamic actions, you will have instances with the same `type` but different `ids`.

Reply:
Removed comment.